### ISSUE TITLE
Migrate to a maintained fork of discord.py

### ISSUE THREAD

**Issue by FM-17** [FM-17/poglink#55]
Now that discord.py is [no longer maintained](https://gist.github.com/Rapptz/4a2f62751b9600a31a0d3c78100287f1), it would be worth migrating to an alternative library.

Suggested Requirements:
- Fork of discord.py 2.x+
- Slash commands (message content is to become [privileged intent](https://support-dev.discord.com/hc/en-us/articles/4404772028055-Message-Content-Privileged-Intent-for-Verified-Bots), slash commands will be standard)
- [Components](https://discord.com/developers/docs/interactions/message-components) (Buttons, dropdown menus, etc)

Initial findings:
- [Reddit Thread](https://www.reddit.com/r/Discord_Bots/comments/plngq0/comment/hcbp9fa/?utm_source=share&utm_medium=web2x&context=3) on discord.py alternatives
- So far, [pycord](https://github.com/Pycord-Development/pycord) meets all requirements. 
- [Pycord vs nextcord](https://www.libhunt.com/compare-pycord-vs-nextcord)

**Comment by travipross** [comment]
[Update](https://gist.github.com/Rapptz/c4324f17a80c94776832430007ad40e6) from discord.py author. 

TL;DR: This issue may evolve into simply upgrading to a newer version of the current library, rather than migrating to a different fork.
